anthos-multi-cloud/Azure/provider.tf

```diff
@@ -34,9 +35,8 @@ provider "azurerm" {
   features {}
 }
 
-provider "azuread" {
-  version = ">= 2.14.0"
-}
+provider "azuread" {}
```

Comment:
Is this block required?

Reply:
It's optional but good-to-have, as customers can add their default Azure Active Directory information here like tenant_id or client_id.

Reply:
Thanks! - I'd suggest adding a comment to that effect.

Reply:
Thank you @VeraQin . This change is unrelated to the main work in this PR. I had to fix it as part of the linter. 
@apeabody - Id a comment explaining what @VeraQin had shared.

Reply:
Done in https://github.com/GoogleCloudPlatform/anthos-samples/pull/593/commits/992d08b6cccc1eaab22cfd55ac4eb3fc55ffb4de